mongoriver.gemspec

```diff
@@ -16,8 +16,8 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = Mongoriver::VERSION
 
-  gem.add_runtime_dependency('mongo', '>= 1.7')
-  gem.add_runtime_dependency('bson_ext')
+  gem.add_runtime_dependency('mongo', ['>= 1.12.5', '< 2.0'])
+  gem.add_runtime_dependency('bson_ext', ['>= 1.12.5', '< 2.0'])
```

Comment:
Because of breaking changes in  mongo 2.0. This code was written in 2014, when I believe mongo2.0 did not exist, so I took that latest 1.12 versions, which actually work.